spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1657,6 +1670,54 @@
             end
           end
         end
+
+        describe 'the profile has deactivation_reason set to encryption_error' do
+          context 'the enrollment passes proofing with USPS' do
+            let(:pending_enrollment) do
+              create(
+                :in_person_enrollment, :pending, :with_notification_phone_configuration
+              )
+            end
+
+            before do
+              allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+              stub_request_passed_proofing_results
+            end
+
+            it 'does not overwrite the deactivation_reason' do
```

Comment:
This description needs to be updated to `overwrites the deactivation_reason`

Reply:
or `sets the deactivation reason to nil`

Reply:
I liked `overwrites the deactivation_reason`. Once we get [LG-14591](https://cm-jira.usa.gov/browse/LG-14591) it will not overwrite- so I stuck with that theme. See commit d6218a12b42cfb5e6537712d8b67f4acf0e036b9